contribs/gmf/src/search/component.js

```diff
@@ -993,6 +1009,10 @@ class SearchController {
             const silent = !!featureGeometry;
             this.gmfTreeManager_.addGroupByLayerName(actionData, true, silent);
           }
+        } else {
+          if (this.executeSearchAction_) {
+            this.executeSearchAction_(action);
+          }
```

Comment:
With es6 arrow function we should be able to call ``searchActionCallback`` (the ``executeSearchAction_`` seem not be necessary) directly, and here:
https://github.com/camptocamp/baselstadt_mapbs/blob/ff51369ec2ab256738e2d51aeb83580614708d3f/mapbs/static-ngeo/js/ngeodesktop.js#L543 the intermediate function can be removed.